client/src/App/index.jsx

```diff
@@ -2,8 +2,8 @@ import React, { useEffect, useState } from 'react';
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import axios from 'axios';
 import { Layout } from 'antd';
+import { Navbar, MainFooter } from '../Components';
```

Comment:
Because I have used Footer component from antd, I can't use two componot with the same name.
I change MainFooter to Footer, and use Paragraph ant component for copyright.

Reply:
That's at the component, but in the `App` you can import the `MainFooter` as `Footer` with no problems